src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -73,22 +98,38 @@ class Artworks extends React.Component<Props> {
   }
 }
 
-export default createPaginationContainer(
+export default createRefetchContainer(
   Artworks,
   {
     filtered_artworks: graphql`
       fragment Artworks_filtered_artworks on FilterArtworks
         @argumentDefinitions(
-          count: { type: "Int", defaultValue: 10 }
-          cursor: { type: "String", defaultValue: "" }
+          first: { type: "Int", defaultValue: 10 }
+          after: { type: "String", defaultValue: "" }
         ) {
         __id
-        artworks: artworks_connection(first: $count, after: $cursor)
-          @connection(key: "Artworks_filtered_artworks") {
+        artworks: artworks_connection(first: $first, after: $after) {
           pageInfo {
             hasNextPage
             endCursor
           }
+          pageCursors {
+            around {
+              cursor
+              page
+              isCurrent
+            }
+            first {
+              cursor
+              page
+              isCurrent
+            }
+            last {
+              cursor
+              page
+              isCurrent
```

Comment:
I haven’t actually tried this with Relay Modern (I don’t think), but maybe you could use a fragment to DRY up these repetitive selections.

Reply:
Yea, I mentioned above, but this definitely can just be moved into its own component and fragment container (w/ fragment there as well).

Reply:
Oh I thought you meant the entire pagination widget.